cmd/revpn.go

```diff
@@ -47,15 +48,23 @@ func start() (*exec.Cmd, error) {
 
 	stdout, err := command.StdoutPipe()
 	if err != nil {
-		return nil, errors.Wrap(err, "stdout pipe error")
+		return nil, errors.Wrap(err, "couldn't get stdout pipe")
+	}
+	stderr, err := command.StderrPipe()
+	if err != nil {
+		return nil, errors.Wrap(err, "couldn't get stderr pipe")
 	}
 	if err := command.Start(); err != nil {
 		return nil, errors.Wrap(err, "netExtender command failed")
 	}
 
-	scanner := bufio.NewScanner(stdout)
+	merged_output := io.MultiReader(stderr, stdout)
+	scanner := bufio.NewScanner(merged_output)
 	for scanner.Scan() {
-		fmt.Println(scanner.Text())
+		fmt.Println("[netExtender] ", scanner.Text())
+		if strings.HasPrefix(scanner.Text(), "Another NetExtender instance is already running") {
+			return nil, errors.New("NetExtender already running; please kill it and try again.")
+		}
```

Comment:
Rather than having a double scanner https://golang.org/pkg/os/exec/#Cmd.CombinedOutput should be used in this case it would simplify the approach quite a bit.

Reply:
I am not convinced this will work, as `CombinedOutput` will only return once command has run to completion, which is not the case with our netExtender

Reply:
oh I see.